docs/how/updating-datahub.md

```diff
@@ -5,6 +5,7 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 ## Next
 
 ### Breaking Changes
+-  #5451 `GMS_HOST` and `GMS_PORT` environment variables deprecated in `v0.8.39` have been removed
```

Comment:
Can you write it what are the environment variables which deprecates this?
Just to make it easy to migration if somebody bumps into this.